src/client/pages/Offer/Checkout/index.tsx

```diff
@@ -258,16 +261,21 @@ export const Checkout = ({
   const privacyPolicyLink = mainQuote.insuranceTerms.find(
     ({ type }) => type === InsuranceTermType.PrivacyPolicy,
   )?.url
+
   const formik = useFormik<QuoteInput>({
```

Comment:
Perhaps investigate how we can allow user to submit form to get benefits from browser form validation. Can we use `checkValidity` with formik:

https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation

Reply:
https://hedvig.atlassian.net/browse/GRW-705